src/main/java/com/webonise/weatherapp/WeatherApplication.java

```diff
@@ -0,0 +1,25 @@
+package com.webonise.weatherapp;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.support.PropertySourcesPlaceholderConfigurer;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+
+@EnableJpaAuditing
+@SpringBootApplication
+public class WeatherApplication {
+
+  public static void main(String[] args) {
+    SpringApplication.run(WeatherApplication.class, args);
+  }
+
+  @Bean
+  public PropertySourcesPlaceholderConfigurer propertySourcesPlaceholderConfigurer() {
+    PropertySourcesPlaceholderConfigurer properties = new PropertySourcesPlaceholderConfigurer();
+    properties.setLocation(new FileSystemResource("/home/webonise/application.properties"));
```

Comment:
why do you want Bean for Property place holder? Spring boot has out of the box configuration for properties file

Reply:
I have placed the database credentials out of the repository so that i don't need to make changes in property file in case of credentials changes. That's why i have preferred to give the path of the file where it is located using Bean.

Reply:
Yes, Spring boot has very good configuration setting for that  `--spring.config.location` you can configure in tomcat from which location you want to pick properties file. So no need to do anything in the code for that.
https://docs.spring.io/spring-boot/docs/1.0.1.RELEASE/reference/html/boot-features-external-config.html